static/js/compose_actions.js

```diff
@@ -155,7 +162,17 @@ export function update_narrow_to_recipient_visibility() {
             $(".narrow_to_compose_recipients").show();
             return;
         }
+    } else if (message_type === "private") {
+        const $to_text = $(".to_text");
```

Comment:
We should scope this selector more precisely (scoping it inside the compose component, for example); one could imagine a world where we had another `to_text` component in the app.

Reply:
In the most recent commit I prepended the `#private-message` selector AND I did the same for the `#stream-message` selector since now they share the `narrow_to_compose_targets` class for the hover/tooltip behavior

Reply:
Yeah, this looks much improved.